### ISSUE TITLE
mark_new_version.sh STILL can't get past precommits on release-1.0 branch

### ISSUE THREAD

**Issue by zmerlynn** [GoogleCloudPlatform/kubernetes#12037]
This is still (now? still?) busted, and is a P(-1), because it's STILL blocking me from #11941. Please, please, can we get the docs working on the release branch?

```
+++ [0730 13:03:26] Building apiserver
+++ [0730 13:03:26] Building go targets for linux/amd64:
    cmd/kube-apiserver
+++ [0730 13:03:34] Placing binaries
etcd -data-dir /tmp/test-etcd.TQhhZC --bind-addr 127.0.0.1:4001 >/dev/null 2>/dev/null
Waiting for etcd to come up.
+++ [0730 13:03:36] On try 1, etcd: : 
{"action":"set","node":{"key":"/_test","value":"","modifiedIndex":3,"createdIndex":3}}
+++ [0730 13:03:37] Starting kube-apiserver
+++ [0730 13:03:37] On try 2, apiserver: : ok
+++ [0730 13:03:37] Updating 
    build/../hack/../api/swagger-spec
+++ [0730 13:03:37] SUCCESS
+++ [0730 13:03:38] Clean up complete
Checking that it builds... OK

Checking for files that need gofmt... OK

Checking for files that need boilerplate... OK

Checking for API descriptions... OK

Checking for docs that need updating... FAIL: changes needed but not made due to --verify
ERROR!
Some docs are out of sync between CLI and markdown.
To regenerate docs, run:
  hack/run-gendocs.sh

Checking for conversions that need updating... OK

Checking for deep-copies that need updating... OK

Checking for swagger spec that need updating... OK

Aborting commit
```

**Comment by zmerlynn** [comment]
cc @nikhiljindal @thockin

**Comment by zmerlynn** [comment]
If I look at what `hack/run-gendocs.sh` wants to do (i.e. the verification failure ... I force-committed the failed doc-versionize to see), this is what I see:

```
diff --git a/docs/user-guide/kubectl/kubectl.md b/docs/user-guide/kubectl/kubectl.md
index f8eebb8..86f34b3 100644
--- a/docs/user-guide/kubectl/kubectl.md
+++ b/docs/user-guide/kubectl/kubectl.md
@@ -1,8 +1,35 @@
 <!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
 
+<!-- BEGIN STRIP_FOR_RELEASE -->
 
-<!-- END MUNGE: UNVERSIONED_WARNING -->
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/user-guide/kubectl/kubectl.md).
 
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
 ## kubectl
 
 kubectl controls the Kubernetes cluster manager
@@ -72,9 +99,7 @@ kubectl
 * [kubectl stop](kubectl_stop.md)       - Gracefully shut down a resource by name or filename.
 * [kubectl version](kubectl_version.md)         - Print the client and server version information.
 
-###### Auto generated by spf13/cobra at 2015-07-21 06:57:19.126137441 +0000 UTC<!-- BEGIN MUNGE: IS_VERSIONED -->
-  <!-- TAG IS_VERSIONED -->
-  <!-- END MUNGE: IS_VERSIONED -->
+###### Auto generated by spf13/cobra at 2015-07-30 20:06:00.815147302 +0000 UTC
 
 
```

**Comment by nikhiljindal** [comment]
Could it be because we just cherrypicked the changes to script, but not the resulting doc changes?
My change, does change kubectl docs by removing the versioned warnings from them.
So if you just run that script once on the docs, then the next time these pre-commits should pass?

**Comment by zmerlynn** [comment]
Let me try that.

**Comment by thockin** [comment]
You're running this from the release branch?  I just ran it and SOMETHING
is weird, looking, too.

**Comment by zmerlynn** [comment]
Is there any reason not to cherry pick #11622, etc. and make the `build` dir the same as `master`?

**Comment by zmerlynn** [comment]
@thockin: Yes.

**Comment by nikhiljindal** [comment]
I didnt cherrypick 11622 assuming that we are going to cut 1.1 soon.
Will send a cherrypick request now.

**Comment by zmerlynn** [comment]
@nikhiljindal: I'm working on it.

**Comment by thockin-cc** [comment]
I'm working on a manual fixup on release-1.0 - we can not change the import
scripts and not actually fixup the docs.  Sigh.

**Comment by zmerlynn** [comment]
@thockin: See my last on #12040. Looks like a cherry pick fixup works.

**Comment by zmerlynn** [comment]
Well, I'll have to propose a PR with that fixup, but still.

**Comment by thockin** [comment]
No, it still needs "manual love" (best band name ever?)

**Comment by zmerlynn** [comment]
Fixed by a combination of #12044 and #12040. Thanks to @thockin and @nikhiljindal for jumping on it.
